packages/frontend-shared/cypress/e2e/e2ePluginSetup.ts

```diff
@@ -155,13 +155,17 @@ async function makeE2ETasks () {
       sinon.stub(ctx.actions.electron, 'openExternal')
       sinon.stub(ctx.actions.electron, 'showItemInFolder')
 
+      const operationCount: Record<string, number> = {}
```

Comment:
Could also use `Map` - `new Map<string, string>`. Not sure which is more idiomatic - just an optional suggestion.

Reply:
Do we ever need to reset the call counts?

Reply:
it is going to reset before each test